app/src/main/res/values/strings.xml

```diff
@@ -83,8 +83,8 @@
   <string name="patiententry_error_both_dateofbirth_and_age_empty">Enter patient’s age</string>
   <string name="patiententry_error_invalid_dateofbirth">Enter patient’s date of birth in DD/MM/YYYY</string>
   <string name="patiententry_error_dateofbirth_is_in_future">Fix patient’s date of birth</string>
-  <string name="patiententry_error_phonenumber_length_less">Phone number cannot be less than 6 digits</string>
-  <string name="patiententry_error_phonenumber_length_more">Phone number cannot be more than 12 digits</string>
+  <string name="patiententry_error_phonenumber_length_less">Phone number cannot be less than %1$s digits</string>
+  <string name="patiententry_error_phonenumber_length_more">Phone number cannot be more than %1$s digits</string>
```

Comment:
Changing these strings will also break `PatientEditScreen` since it depends on these resources as well. Can we update that screen to account for the differences in number limits in countries as well?